source/js/components/Dashboard/ResultView/ResultList.jsx

```diff
@@ -144,7 +143,8 @@ class ResultList extends Component {
             { sortedItems.map((item) => (
               <ResultListItem
                 item={ item }
-                categoryColor={ getCategoryColor(item) }
+                categoryColor={ getCategory(item).get('color') }
+                editLinkText={ getCategory(item).get('editLinkText') }
```

Comment:
we should not call getCategory twice, as we will trigger the find-action again. We should cache the property inside sortedItems.map.